mbq/atomiq/producers.py

```diff
@@ -49,8 +49,36 @@ def _is_running_within_transaction(self):
         save points and expect there at least to be 2: 1 from TestCase and 1 user-defined.
         """
         if RUNNING_TESTS:
-            if len(db_connection.savepoint_ids) < 2:
-                return False
+            import inspect
```

Comment:
There's no reason to import this inline

Reply:
I want to hide everything related to this piece of code in one nice pile so that we never have to be reminded of it when looking it the rest of the code.

Reply:
No dude import inlines fucking suck, we've lost the battle in OS Core already but there's no reason to do it when we don't need to

Reply:
Going to hold the line hard on this in new repos, and this is a new repo, so no. The Django thing is one exception forced on us by Django's dumbness, but again it does not affect stdlib imports like `inspect`

Reply:
urg fine